certificate.go

```diff
@@ -90,33 +92,27 @@ func addCertToMachine(userCertDecode string, CERTUTIL_PROGRAM_PATH string) error
 								if exitErr, ok := err.(*exec.ExitError); ok {
 									if status, ok := exitErr.Sys().(syscall.WaitStatus); ok {
 										exitStatus := status.ExitStatus()
-										log.Print("Exit Status: ", exitStatus)
+										addNewLinesToDebug("Exit Status: " + string(exitStatus))
 										switch exitStatus {
 										case int(ERROR_INVALID_PASSWORD):
-											walk.MsgBox(windowMsgBox, T("errorWindowTitle"), T("wrongPassword"), walk.MsgBoxOK)
+											addNewLinesToDebug(T("wrongPassword"))
+											viewErrorAndExit(T("wrongPassword"))
 											badCertificatePassword = true
 											mw.Close()
 										case int(ERROR_INVALID_DATA):
-											walk.MsgBox(windowMsgBox, T("errorWindowTitle"), T("invalidCertificate"), walk.MsgBoxOK)
-											os.Remove(userCertDecode)
```

Comment:
I agree that this massive copy/paste needs to be removed but I don't see it in the defer so that it still happens when this is done

This would mean the files are left there after it exits unless I'm missing where this has been added

Reply:
All files are now removed on exit.
In utils.go, func cleanTmpFiles